packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -34,6 +34,14 @@ export interface SignInAliases {
    * @default false
    */
   readonly preferredUsername?: boolean;
+
+  /**
+   * Whether sign-in aliases should be evaulated with case sensitivity.
+   * For example, when this option is set to False, users will be able to sign in using either "username" or "Username".
```

Comment:
```suggestion
   * For example, when this option is set to false, users will be able to sign in using either `username` or `Username`.
```

Reply:
might be a good idea to use an example that doesn't use the same name as a parameter (`username`) to avoid confusion